### ISSUE TITLE
Add cmd for printing object store config format for each provider

### ISSUE THREAD

**Issue by bwplotka** [improbable-eng/thanos#572]
This is to make sure it is clear that it is easy to configure bucket for Thanos.

**Comment by jturolla** [comment]
Maybe this could be solved with better documentation instead?

**Comment by bwplotka** [comment]
Good point, but I think both. In the same way you have both docs and flags in `--help`.

It's easier to print help and see corresponded yaml schema we support with the binary you just use, then tracking down what is the version of your thanos binary, where are the docs, changing to corresponded tag and checking the schema, isn't it? (:

**Comment by jturolla** [comment]
Agreed.

**Comment by FUSAKLA** [comment]
@bwplotka I can take a look at this if you could give any more details on this.

What should the cmd look like? I have no idea how to name it for example `./thanos config object-storage` ?

Also do you want to print all config examples by one command or allow to specify exact os?
Possibly add validation also same way as pomtool or amtool?

**Comment by bwplotka** [comment]
Thanks @FUSAKLA 

So what we have right now is: https://github.com/improbable-eng/thanos/pull/645
We document the format and autogenerate it if needed. 

**The acceptance criteria for this one would be:** 
- Allow validatation (nice idea @FUSAKLA ) by some `thanos bucket config-validate` command.
- Print configuration `thanos bucket config <objectstore>` per tool.

I feel like we should not focus on this until it would be really confirmed it is needed, as we have autogenerated docs now. We have more important tickets to do (: Closing this for now.
